### DISCUSSION TITLE
External hyperlinks to Wiki.JS pages fail

### DISCUSSION THREAD

**Discussion by DaveTahseen** [requarks/wiki#6837] [Help / Questions]
Hi,

I'm struggling to get hyperlinks to work linking to Wiki articles from e.g. a Word or Excel document.
E.g. in Word I can make a hyperlink to https://wiki.xxx.xxx/ which is running on our internal network, this works. The browser opens and shows the Wiki page.

However If I link to e.g. https://wiki.xxx.xxx/en/mypage this fails. Word or Excel return with an error "Cannot Download The Information You Requested" There are many suggestions mostly pointing to the official fix from [Microsoft](https://answers.microsoft.com/en-us/msoffice/forum/all/office-19-hyperlink-returns-cannot-download/4fe351a8-6438-4167-b008-7045a0ce1971) but this is not a solution.

It looks like Excel/Word only gets a 200 response from the web server's Wiki root address https://wiki.xxx.xxx/ not from any sub page address.
I ran a test using some [random VBA code ](https://www.mrexcel.com/board/threads/check-if-url-exists.639463/) and it seems any other url doesn't return http 200 ok.
https://wiki.xxx.xxx/				302 - Found
https://wiki.xxx.xxx/en/mypage		403 - Forbidden
https://wiki.xxx.xxx/en/mypage/	301 - Moved Permanently

Is this the intended behavior?
Unfortunately I have no access to the Wiki server itself so I'm limited to client side information and tests only.

Thanks in advance.

**Comment by NGPixel** [comment]
I'm assuming your wiki is private and as such requires authentication. Word is probably loading the pages without authentication, which is why you're getting errors, but not in your browser.
